app/views/foods/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1 class='food-item'>Add new food Item</h1>
+<%= form_with(model:@food,url: foods_path, method: :post,local: true,class:'food-form') do |form| %>
+    <%= form.label:name, "Food name:", class:"form-label"%>
+    <%= form.text_field:name %>
+    <%= form.label:measurement_unit, 'Measrement unit' ,class:"form-description" %>
+    <%= form.text_field:measurement_unit %>
+    <%= form.label:price, 'Price' ,class:"form-description" %>
+    <%= form.text_field:price %>
+    <%= form.label:quantity, 'Quantity' ,class:"form-description" %>
+    <%= form.text_field:quantity %>
+    <%= form.submit 'Add food' ,class:"form-submit" %>
+<% end %>
```

Comment:
- [ ] [OPTIONAL] While a design for this was not explicitly provided, it would be great if you added some styling to this form. This is how it looks at the moment:

![image](https://github.com/ismayito/Rails-recipe-app/assets/86318284/d8dee0de-688e-409a-80eb-9f7e9b4cc3d8)

Reply:
Fixed. Added styling based on the suggestion 
![image](https://github.com/ismayito/Rails-recipe-app/assets/93598189/d634f232-0bee-4f57-96e4-599c6fb7a64f)